src/main/java/me/zeroeightsix/kami/module/modules/misc/AntiAFK.kt

```diff
@@ -39,9 +39,11 @@ internal object AntiAFK : Module(
     private val walk = setting("Walk", true)
     private val radius by setting("Radius", 64, 8..128, 8)
     private val inputTimeout by setting("InputTimeout(m)", 0, 0..15, 1)
+    private val allowBreak by setting("AllowBreak", false, { walk.value })
```

Comment:
honestly in something like AntiAFK this should be called AllowBreakingBlocks

Reply:
sorry for all the requested changes

Reply:
Don't worry, they are small changes that I can do quickly, and I understand that you want to keep the project as nice as possible, I would as well in your situation.